lib/src/main.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:first_flutter_app/src/screen/dice/dice_screen.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatefulWidget {
```

Comment:
Lets get the App widget in a separate file, and just use this to run it, in this way you will be able to have different launch targets in the future

Reply:
I created a separate file to app widget. Thanks!